cabal-install/src/Distribution/Client/ProjectBuilding.hs.orig

```diff
@@ -0,0 +1,2345 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
Is this .orig file committed by mistake?

Reply:
Yes, it is. git creates these when resolving conflicts with `git mergetool -t vimdiff3`...
I've removed and pushed again. I'm assigning to merge.